pkgs/top-level/all-packages.nix

```diff
@@ -20162,6 +20162,8 @@ with pkgs;
 
   surge-cli = callPackage ../development/tools/surge-cli { };
 
+  surrealist = callPackage ../development/tools/surrealist { };
```

Comment:
move the default.nix file to pkgs/by-name/su/surrealist/package.nix

Reply:
Moved it :+1: